### ISSUE TITLE
ImpactFrames Can't find ollama models at "selected_model"

### ISSUE THREAD

**Issue by HyperUpscale** [if-ai/ComfyUI-IF_AI_tools#38]
I am looking for information, but there is none about this...

Even if I take ready working Workflow - with exact same setup: Windows Ollama (WSL or native) - even the same local addresses, ports, same models ... 

I tired different nodes that works with my Ollama and they all work without a problem.

I tried the 3 of the ImpactFrames LLM related, but none of them seem to get the Ollama models - the field is not active: 

![image](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/4c050569-d4ac-4e50-8899-0bad84cca2f5)

![image](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/0254d2da-d5ca-484e-baec-0606c2b977f3)

![image](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/086c9a66-8980-49fb-8bc7-550eb170ffaf)

Any Ideas how to troubleshoot this?

**Comment by if-ai** [comment]
did you tried with 127.0.0.1 as your base ip? perhaps you are using a vpn that is blocking the localhost otherwise should work as long as you have ollama running and no other app using the same port 11434
![ProtonVPN_HESBgYjzlT](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/21185218/26bba85a-26c1-4569-838b-752f821a6934)

**Comment by HyperUpscale** [comment]
Ok... thats a great suggestion 
![image](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/e2b8bbae-7668-4198-9b8a-fda7976a22de)

But let me try to explain again - other Ollama tools are working... ONLY IF tool is not.
that means - there is no problem with IP, firewall, or base URL...

Do you want to try to advice me how can I check log file or something to find the problem... I want to skip the back and forth plain suggestion - I don't want to lose your time. I want to find by myself.

**Comment by HyperUpscale** [comment]
Alright... I kinda fixed it Manually. Which Proves there is something wrong with the node... At least for me.

So manual set of the widget value fixes it :)

      "widgets_values": [
        "a flower",
        "127.0.0.1",
        "11434",
        "ollama",
        **"impactframes/llama3_ifai_sd_prompt_mkr_q4km:latest",**
        "IF_PromptMKR",
        "",


![image](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/5b66687a-b676-46de-8cac-281c1b66917b)


And WHEN I click the arrow on the Node to swap the model... it goes to undefined again ... which causes the json in line for the model to become NULL...

Just for your information. 
I will adjust manually the json, but something is not OK :) 

Greetings.

**Comment by HyperUpscale** [comment]
Just a note:
The other Node that I tried (not IF) also doesn't query the models...
But it is strange that when I Ques the prompt, then they got queried.
 Maybe it is normal... I don't know 
 
 WHEN I Set manual Wrong value , that is not Null I see this error:
 
![step 1](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/aeb1cca8-119e-4106-b70b-b7c1b8d9139e)

This is the correct reply form "/api/tags" query (the lint of the available models) 
![image](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/3b1922f6-9e8a-48b1-b7b1-3a16d38cb077)

**Comment by HyperUpscale** [comment]
BTW .. maybe I am doing something wrong... As I just found out that I have the same issue with my stable diffusion ckpt images... which has nothing to do with IF, nor Ollama, nor IP address ... they are local files. 

It is possible the problem to be from StableSwarmUI, which I am using...

I see also the same case error for my new workflow, trying to use Checkpoint loader...

![image](https://github.com/if-ai/ComfyUI-IF_AI_tools/assets/126105457/341a8e3b-feea-4577-b3bd-0f654a92a791)
    
  So I think we can close this one.
  
 Sorry for  the issue.
